docs/scripts_md/remove_jiv_data_from_db_and_filesystem.pl

```diff
@@ -0,0 +1,22 @@
+# NAME
```

Comment:
why is the `.pl` file in the docs/scripts_md directory?

Reply:
I guess I forgot to change the extension .md when creating the MD file...

Reply:
fixed now